frontend/src/app/components/charts/CompaniesPieChart.tsx

```diff
@@ -1,14 +1,18 @@
 import { ApexOptions } from "apexcharts";
-import React from "react";
+import React, { useState } from "react";
 import ReactApexChart from "react-apexcharts";
 
+interface CompaniesPieChartProps {
+  companyTiers: number[];
+}
+
 const options: ApexOptions = {
   chart: {
     fontFamily: "Satoshi, sans-serif",
     type: "donut",
   },
-  colors: ["#62e371", "#40c74f", "#2cb03b", "#158f23"],
-  labels: ["521-560", "501-520", "246-500", "0-245"],
+  colors: [ "#4BAE50", "#CCDA38", "#FEC005", "#FE5620"],
```

Comment:
I'm wondering if we can consider alternate colors for red-green color blindness for accessibility purposes. Maybe a blue-orange scale? Idk tho, the red-green is more intuitive in general